### PR TITLE
Feature/adarmenov/ignio

### PR DESCRIPTION
**Pull request by adarmenov**
IGNI is a new component that integrates fire weather observations into fuel moisture codes and fire behavior indexes based on the CFFWI system.

### CODE REVIEW

File: GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOS_LandGridComp.F90 [hunk 1]
```diff
@@ -204,7 +207,14 @@ subroutine SetServices ( GC, RC )
 !          end do
 !       end if
 !    ENDIF
-    
+   
+    if (DO_FIRE_DANGER /= 0) then
```
Comment (sanAkel) [inline GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOS_LandGridComp.F90:211]: I do not favor such integer criteria. Instead recommend logical `.true.` or `.false.`

**Review by sanAkel — CHANGES_REQUESTED** [review]
Reviewing on behalf of @GEOS-ESM/ocean-team

**Comment by gmao-rreichle** [comment]
Based on a brief inspection of the changes this looks ok to me from the land perspective.  
To be safe, @biljanaorescanin should run the standard GEOSldas tests before the @GEOS-ESM/land-team approves.
(Also needs updating to the latest develop branch.)

File: GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOS_LandGridComp.F90 (hunk 1 above)
Comment (adarmenov) [reply GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOS_LandGridComp.F90]: Addressed with https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/699/commits/df4695f0b650d240deb95292a96bed12a0267e27

**Review by zhaobin74 — APPROVED** [review]
Not related to landice. Approve.

**Comment by adarmenov** [comment]
> ... (Also needs updating to the latest develop branch.)

@gmao-rreichle Addressed with  https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/699/commits/04821aa33a372867eaf9dca8e2e32a3df4f3cbe0

**Review by tclune — APPROVED** [review]
Please note that CMake provides a mechanism to register tests, and thereby make it easier for us to incorporate them into CI testing.   It would also avoid hardcoding the compiler (gfortran in this case)

 If at all interested please contact @mathomp4 or myself.

**Review by sanAkel — APPROVED** [review]

**Comment by sanAkel** [comment]
> Please note that CMake provides a mechanism to register tests, and thereby make it easier for us to incorporate them into CI testing. It would also avoid hardcoding the compiler (gfortran in this case)
> 
> If at all interested please contact @mathomp4 or myself.

Yes, I'm interested and will reach out to @mathomp4. Thanks!

**Review by biljanaorescanin — APPROVED** [review]

**Review by sdrabenh — APPROVED** [review]

**Pull request merged**